### ISSUE TITLE
XUNITReporter failure after 2.15

### ISSUE THREAD

**Issue by BenPatterson2** [testem/testem#1336]
We just started getting an error after the last release

Testem settings
```

  report_file: 'tests/test-results.xml',
  reporter: 'xunit',
  xunit_intermediate_output: true,
  parallel: '4',
```

Failure
```
1..2241
# tests 2241
# pass  2223
# skip  16
# fail  2
/src/node_modules/bluebird/js/release/using.js:12
        setTimeout(function(){throw e;}, 0);
                              ^

TypeError: text.split is not a function
    at indent (/src/node_modules/testem/lib/strutils.js:13:15)
    at XUnitReporter.getTestResultNode (/src/node_modules/testem/lib/reporters/xunit_reporter.js:87:25)
    at XUnitReporter.summaryDisplay (/src/node_modules/testem/lib/reporters/xunit_reporter.js:56:31)
    at XUnitReporter.finish (/src/node_modules/testem/lib/reporters/xunit_reporter.js:40:25)
    at reporters.forEach.reporter (/src/node_modules/testem/lib/utils/reporter.js:126:22)
    at Array.forEach (<anonymous>)
    at Reporter.finish (/src/node_modules/testem/lib/utils/reporter.js:124:20)
    at Reporter.close (/src/node_modules/testem/lib/utils/reporter.js:70:10)
    at Reporter.Bluebird.try.disposer (/src/node_modules/testem/lib/utils/reporter.js:114:19)
    at FunctionDisposer.doDispose (/src/node_modules/bluebird/js/release/using.js:98:19)
    at FunctionDisposer.Disposer.tryDispose (/src/node_modules/bluebird/js/release/using.js:78:20)
    at iterator (/src/node_modules/bluebird/js/release/using.js:36:53)
    at dispose (/src/node_modules/bluebird/js/release/using.js:48:9)
    at /src/node_modules/bluebird/js/release/using.js:194:20
    at PassThroughHandlerContext.finallyHandler (/src/node_modules/bluebird/js/release/finally.js:56:23)
    at PassThroughHandlerContext.tryCatcher (/src/node_modules/bluebird/js/release/util.js:16:23)
```

It looks related to 
https://github.com/testem/testem/commit/5ec77becb21622d78e50653a87c80436d4115af2#diff-b60cc068c57ea8ae2c2b8bf84fe5cfe7R87

This appears to be when Non strings are used in expects. The test that is causing failure is akin to 
`   assert.equal("Fake", false);`

**Comment by BenPatterson2** [comment]
Will have a PR up for this in a few

**Comment by johanneswuerbach** [comment]
Thank you, #1336 released in `v2.15.1`
